src/Select/index.js

```diff
@@ -102,50 +105,78 @@ const Select = ({
   } = useSelect(downshiftOpts);
 
   const hasSelectedItem = selectedItem !== undefined && selectedItem.props;
+  const showMenu = isOpen && items.length > 0;
+
+  /** @see https://github.com/everweij/react-laag#api-docs */
+  const { renderLayer, triggerProps, triggerBounds, layerProps, layerSide } =
+    useLayer({
+      isOpen: showMenu,
+      auto: true,
+      snap: true,
+      triggerOffset: 0,
+      containerOffset: 0,
+      placement: "bottom-start",
+      possiblePlacements: ["top-start", "bottom-start"],
+    });
 
   return (
     <div className="nds-select" data-testid={testId}>
       <DropdownTrigger
-        isOpen={isOpen}
+        isOpen={showMenu}
         labelText={label}
         displayValue={getSelectedItemDisplay(selectedItem)}
         labelProps={{ ...getLabelProps() }}
         errorText={errorText}
-        {...getToggleButtonProps()}
+        {...getToggleButtonProps(triggerProps)}
```

Comment:
This is how we combine the `react-laag` element props with the `downshift` prop getters.

All the prop getters like this from `downshift` take an argument; an object of additional props to apply.